docs/src/app/components/pages/components/IconMenu/Page.jsx

```diff
@@ -13,12 +13,15 @@ import IconMenuExampleControlled from './ExampleControlled';
 import iconMenuExampleControlledCode from '!raw!./ExampleControlled';
 import IconMenuExampleScrollable from './ExampleScrollable';
 import iconMenuExampleScrollableCode from '!raw!./ExampleScrollable';
+import IconMenuExampleNested from './ExampleNestedDrop';
+import iconMenuExampleNestedCode from '!raw!./ExampleNestedDrop';
 
 const descriptions = {
   simple: 'Simple Icon Menus demonstrating some of the layouts possible using the `anchorOrigin` and `' +
   'targetOrigin` properties.',
   controlled: 'Two controlled examples, the first allowing a single selection, the second multiple selections.',
   scrollable: 'The `maxHeight` property limits the height of the menu, above which it will be scrollable.',
+  nested: 'Example of nested menus within a icon menu.',
 };
```

Comment:
"_an_ Icon Menu" (or "an `IconMenu`") - I'm still never sure which, despite the fact I wrote all those descriptions. :smile: 

Icon Menu to refer to the feature, `IconMenu` to refer to the React component. Probably the latter - I don't think I was consistent.